spring-cloud-aws-dynamodb/src/test/java/io/awspring/cloud/dynamodb/DynamoDbTemplateIntegrationTest.java

```diff
@@ -96,23 +123,27 @@ void dynamoDbTemplate_saveUpdateAndRead_entitySuccessful() {
 		assertThat(personEntity1).isEqualTo(personEntity);
 
 		// clean up
-		cleanUp(personEntity.getUuid());
+		cleanUp(dynamoDbTable, personEntity.getUuid());
 	}
 
-	@Test
-	void dynamoDbTemplate_saveAndDelete_entitySuccessful() {
+	@ParameterizedTest
```

Comment:
I really dislike `@ParameterizedTest`.
Would rather prefer `@Nested`
But I see code deduplication will be there so it is fine.